src/pc/stats.rs

```diff
@@ -118,7 +118,7 @@ pub struct ParachainStats {
 	backed_count: u32,
 	/// Number of skipped slots, where no candidate was backed and availability core
 	/// was free.
-	skipped_slots: u32,
+	skipped_slots: Vec<u32>,
```

Comment:
These have to be the block hashes.

Reply:
I think that block numbers are somehow more readable. Like we have missed a slot on block `6`, `8` and `9` instead of some long hash. But I don't have a strong opinion on that.

Reply:
We can collect them together: `(number, hash)`.

Reply:
Yeah, maybe tuple is best. I usually want the hash so I can use it to filter logs. With numbers I'd still have to translate to hash, and if there are forks it will be more work.

Reply:
Fixed it to tuples

Reply:
Replaced with structs :-)